src/vm/eventpipe.h

```diff
@@ -73,7 +74,15 @@ class EventPipeEventPayload
 
 public:
     // Build this payload with a flat buffer inside
-    EventPipeEventPayload(BYTE *pData, unsigned int length);
+    EventPipeEventPayload(BYTE *pData, unsigned int length) :
```

Comment:
Consistency? Most EventPipe* files have these *LIMITED_METHOD_CONTRACT* in the header files. I can undo, it's no big deal.

Reply:
Its unevenly applied in the runtime but I think of it as:
- Ideal world is that all code is in cpp files unless it has to be in the header (for example inlining or templated code). I'd expect it to improve build time because headers get included in numerous compilation units that won't need that code, compiler and linker do extra work to remove it later.
- Pragmatic world is that devs often put little methods in headers because it saves them time.

Don't worry about this one because it isn't a big deal either way, but in the future you can save time and churn in the review by leaving code in the .cpp.